src/streamed/index.ts

```diff
@@ -205,9 +208,9 @@ export class StreamedOrderbook {
     );
     const confirmedBlock = latestBlock - this.options.blockConfirmations;
 
-    this.batch = await this.getBatchId(latestBlock);
+    this.batch = await this.getBatchId(toBlock ?? latestBlock);
```

Comment:
nanonit: Maybe we should have a `const endBlock = toBlock ?? latestBlock` or something? I'm OK if you prefer it this was as well - I just tend to prefer binding values to variables when they have important values and are used in multiple places.